teuthology/orchestra/console.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, name, ipmiuser, ipmipass, ipmidomain, logfile=None,
         self.has_credentials = all(
             [self.ipmiuser, self.ipmipass, self.ipmidomain]
         )
+        self.conserver_master = config.conserver_master
+        self.conserver_port = config.conserver_port
+        conserver_client_found = subprocess.Popen(
+            'which console',
```

Comment:
1) I'm unsure, actually. I'd love feedback for how to better make this decision
2) Well, it's a binary on f24, centos 7, trusty, and xenial. I haven't yet seen a system without it.